api/app/models/collection.rb

```diff
@@ -50,6 +55,9 @@ class Collection < ApplicationRecord
 
   scope :search_import, -> { includes(:project) }
 
+  # Callbacks
+  after_commit :trigger_creation_event, on: [:create]
```

Comment:
Ensure that there is a spec for this behavior:
Collection enqueues a Event:: COLLECTION_ADDED event on creation

Reply:
✔